### ISSUE TITLE
Focus inconsistancies

### ISSUE THREAD

**Issue by vpicone** [carbon-design-system/gatsby-theme-carbon#773]
- Next/prev missing focus styles
- edit in github link missing focus styles

We should take a sec to audit focus styles throughout the theme default elements.

**Comment by sstrubberg** [comment]
The footer needs focus states as well.

![image](https://user-images.githubusercontent.com/15822070/76332905-e72cc280-62be-11ea-9990-b9906d07544b.png)
